packages/react-components/src/components/VideoGallery/OverflowGallery.tsx

```diff
@@ -96,7 +96,7 @@ export const OverflowGallery = (props: {
         onFetchTilesToRender={onFetchTilesToRender}
         onChildrenPerPageChange={onChildrenPerPageChange}
       >
-        {overflowGalleryElements}
+        {overflowGalleryElements ? overflowGalleryElements : [<></>]}
```

Comment:
Are you doing this so that an empty VerticalGallery will be rendered and push down the local video tile?

Reply:
yup! just more protections against undefined children in the gallery. this was prone to blow up if I didn't add these